data/human/human missions.txt

```diff
@@ -5547,3 +5547,221 @@ mission "Quicksilver Mixup 1"
 			
 
 event "quicksilver mixup 2 available"
+
+
+
+# Rationale for Douglas' actions; in 1000 years, treatment for blindness would be expected to be better. So he is able to do some limited actions, like recognize people, but not much more.
```

Comment:
Was there any specific reason to leave this in? I couldn't think of any case where this popped up.

Reply:
I'm not so sure now, I can delete it.

Reply:
"Blind" doesn't always mean "can't see anything at all." Many blind people do actually have some light sensitivity. You also don't necessarily need to be able to see to recognise a person.

Reply:
```suggestion
```